docs/client-development/mandate.rst

```diff
@@ -0,0 +1,526 @@
+
+    .. code::
+
+        GET /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__natuurlijkPersoon__inpBsn=<own_bsn>
+             &rol__machtiging=machtiginggever
+
+
+    Additionally ``rol.omschrijvingGeneriek`` can be used to determine the "belanghebbende"
+    of the case.
+
+    .. code::
+
+        GET /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__natuurlijkPersoon__inpBsn=<own_bsn>
+             &rol__omschrijvingGeneriek=belanghebbende
+
+
+3. Show me the cases (based on my BSN) where I represent another party:
+
+    .. code::
+
+        GET /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__natuurlijkPersoon__inpBsn=<own_bsn>
+             &rol__machtiging=gemachtigde
+
+
+    It's also possible to filter cases based on the level of assurance and to exclude
+    results where levelOfAssurance is below some required value.
+
+    .. code::
+
+        GET /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__natuurlijkPersoon__inpBsn=<own_bsn>
+             &rol__machtiging=gemachtigde
+             &rol__machtiging__loa=urn:oasis:names:tc:SAML:2.0:ac:classes:MobileTwoFactorContract
+
+
+**eHerkenning**
+
+Filters for eHerkenning authorized parties depend if it's an organization or the branch, therefore
+there are two examples for each option.
+
+1. Show me my cases (based on my KVK nummer) opened for myself:
+
+    .. code::
+
+        # organization
+        GET /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__nietNatuurlijkPersoon__kvk_Nummer=<own_kvk>
+             &rol__machtiging=eigen
+
+    .. code::
+
+        # branch
+        GET /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__vestiging__kvkNummer=<own_kvk>
+             &rol__machtiging=eigen
+
+
+    For organizations it's also possible to filter on their RSIN:
+
+    .. code::
+
+        GET /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__nietNatuurlijkPersoon__innNnpId=<own_rsin>
+             &rol__machtiging=eigen
+
+
+    Additionally ``rol.omschrijvingGeneriek`` can be used to determine the "initiator"
+    of the case
+
+    .. code::
+
+        GET /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__nietNatuurlijkPersoon__kvk_Nummer=<own_kvk>
+             &rol__machtiging=eigen
+             &rol__omschrijvingGeneriek=initiator
+
+
+2. Show me my cases (based on my KVK nummer) opened by an authorizee on my behalf:
+
+    .. code::
+
+        # organization
+        GET /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__nietNatuurlijkPersoon__kvk_Nummer=<own_kvk>
+             &rol__machtiging=machtiginggever
+
+    .. code::
+
+        # branch
+        GET /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__vestiging__kvkNummer=<own_kvk>
+             &rol__machtiging=machtiginggever
+
+
+    Additionally ``rol.omschrijvingGeneriek`` can be used to determine the "belanghebbende"
+    of the case.
```

Comment:
Here too I don't get the semantic difference with the previous example :thinking:

Reply:
Please take a look at this [comment](https://github.com/open-zaak/open-zaak/pull/1736#discussion_r1842349373)